src/main/java/com/google/cloud/tools/maven/deploy/ConfigProcessor.java

```diff
@@ -14,20 +14,18 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.maven.config;
+package com.google.cloud.tools.maven.deploy;
 
-import com.google.cloud.tools.appengine.operations.Gcloud;
-import com.google.cloud.tools.maven.deploy.AbstractDeployMojo;
+import com.google.cloud.tools.maven.cloudsdk.ConfigReader;
 import com.google.common.annotations.VisibleForTesting;
-import java.nio.file.Path;
 
-public class AppYamlConfigProcessor implements ConfigProcessor {
+public class ConfigProcessor {
+  static final String APPENGINE_CONFIG = "APPENGINE_CONFIG";
```

Comment:
`@VisibleForTesting`?

Reply:
I think I should move these back to config reader... must have accidentally left them here when I was playing around with different way to compartmentalize everything.